cmd/cloud/cluster_installation.go

```diff
@@ -245,3 +273,90 @@ var clusterInstallationMattermostCLICmd = &cobra.Command{
 		return nil
 	},
 }
+
+// Command to migrate cluster installation(s)
+var clusterInstallationsMigrationCmd = &cobra.Command{
+	Use:   "migration",
+	Short: "Migrate installation(s) to the target cluster.",
+	RunE: func(command *cobra.Command, args []string) error {
+		command.SilenceUsage = true
+
+		serverAddress, _ := command.Flags().GetString("server")
+		client := model.NewClient(serverAddress)
+
+		sourceCluster, _ := command.Flags().GetString("source-cluster")
+		targetcluster, _ := command.Flags().GetString("target-cluster")
+		installation, _ := command.Flags().GetString("installation")
+
+		client.MigrateClusterInstallation(&model.MigrateClusterInstallationRequest{SourceClusterID: sourceCluster, TargetClusterID: targetcluster, InstallationID: installation, DNSSwitch: false, LockInstallation: false})
```

Comment:
I think I mentioned it earlier but we should handle errors in CLI (there are several places):
```suggestion
		err := client.MigrateClusterInstallation(&model.MigrateClusterInstallationRequest{SourceClusterID: sourceCluster, TargetClusterID: targetcluster, InstallationID: installation, DNSSwitch: false, LockInstallation: false})
		if err != nil {
		         return err
		}
```